salt/states/btrfs.py

```diff
@@ -100,9 +99,9 @@ def wrapper(*args, **kwargs):
                     ret["comment"].append(msg)
                 kwargs["__dest"] = dest
             ret = action(*args, **kwargs)
-        except Exception as e:  # pylint: disable=broad-except
-            log.error("""Traceback: {}""".format(traceback.format_exc()))
-            ret["comment"].append(e)
+        except Exception as exc:  # pylint: disable=broad-except
+            log.error("Exception raised while mounting device: %s", exc, exc_info=True)
```

Comment:
**question** shouldn't this be `log.exception(...)`?

Reply:
It could be, decided to keep the previous log level instead and use `exc_info` to include the traceback...